app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/Helper.java

```diff
@@ -0,0 +1,11 @@
+package edu.drexel.lapcounter.lapcounter.backend;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.v4.content.LocalBroadcastManager;
+
+public class Helper {
```

Comment:
Maybe `BroadcastHelper` or something like that? "Helper" is rather vague. Otherwise this code looks good